code-of-conduct/enforcement-manual.md

```diff
@@ -0,0 +1,66 @@
+This can include contacting law enforcement (or other local personnel) and speaking on behalf of the OBF.
+
+If the act is ongoing, any community member may act immediately, before reaching consensus, to diffuse the situation.
+In ongoing situations, any member may at their discretion employ any of the tools available in this enforcement manual, including bans and blocks online, or removal from a physical space.
+
+In situations where an individual community member acts unilaterally, they must inform the appointed members as soon as possible, and report their actions for review within 24 hours.
+
+## 4.2 Less-Urgent Situations
+
+Upon receiving a report of an incident, the Code of Conduct committee will review the incident and determine, to the best of their ability:
+
+- whether this is an ongoing situation
+- whether there is a threat to anyone's physical safety
+- what happened
+- whether this event constitutes a Code of Conduct violation
+- who, if anyone, was the bad actor
+
+This information will be collected either in person or in writing.
+The Code of Conduct committee will provide a written summary of the information surrounding the incident.
+All participants will be anonymized in the summary report, referred to as "Community Member 1", "Member project 1", or "Board member 1" and so on.
+The "de-anonymizing key" will be kept in a separate file and only accessed to link repeated reports against the same person over time.
+
+The Code of Conduct committee will aim to have a resolution agreed upon within one week.
+If a resolution can't be determined in that time, a member of the Code of Conduct committee will respond to the reporter(s) with an updated and projected timeline for resolution.
+
+## 4.3 Resolutions
+
+The Code of Conduct committee will seek to agree on a resolution by consensus of all members investigating the report in question.
+If the committee cannot reach consensus and deadlocks for over a week, the appointed members from the OBF board will break the tie.
+If these members are unable to take part in the discussion due to a conflict of interest, the chair of the OBF board will make the decision.
+
+Possible responses may include:
+
+* A mediated conversation or agreement between the impacted community members.
+* A request for a verbal or written apology, public or private, from a community member.
+* A public announcement clarifying community responsibilities under the Code of Conduct.
+* Nothing, if the issue reported is not a violation or outside of the scope of this Code of Conduct.
+* A private in-person conversation between a member of the OBF Code of Conduct committee and the individual(s) involved.
+  In this case, the person who has the conversation will provide a written summary for record-keeping.
+* A private written reprimand from a member of the research team to the individual(s) involved.
```

Comment:
What does _research team_ mean?

Reply:
That must be an error, updating to coc committee